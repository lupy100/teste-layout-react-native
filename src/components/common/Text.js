// @flow
import styled from 'styled-components/native';

export default styled.Text`
  font-size: 20;
  text-align: center;
  color: #becce1;
  font-weight: bold;
`;
