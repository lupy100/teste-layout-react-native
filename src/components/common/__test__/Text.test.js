// @flow
import * as React from 'react';
import renderer from 'react-test-renderer';

import Text from '../Text';

test('renders Text correctly', () => {
  const tree = renderer.create(<Text>This is my Text</Text>).toJSON();
  expect(tree).toMatchSnapshot();
});
