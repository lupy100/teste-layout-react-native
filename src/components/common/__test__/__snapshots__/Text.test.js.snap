// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Text correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#becce1",
        "fontSize": 20,
        "fontWeight": "bold",
        "textAlign": "center",
      },
      undefined,
    ]
  }
>
  This is my Text
</Text>
`;
